test/MUXControlsTestApp/App.xaml.cs

```diff
@@ -1,6 +1,9 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+
+#define FEATURE_PAGER_ENABLED // Need to invistigate further on how to remove this line.
```

Comment:
>FEATURE_PAGER_ENABLED [](start = 8, length = 21)

We probably shoudn't check this in, Can we investigate?

Reply:
I am on it

Reply:
Turns out the control works fine without this line!